pkg/imageengine/buildah/build_rootfs.go

```diff
@@ -15,36 +15,11 @@
 package buildah
 
 import (
-	"fmt"
-
 	"github.com/sealerio/sealer/pkg/define/options"
-
-	"os"
 )
 
-// BuildRootfs will make an image rootfs under /var/lib/containers/storage
-// And then link it to the DestDir
-// And remember to call RemoveContainer to remove the link and remove the container(umount rootfs) manually.
+// BuildRootfs need to clarify responsibility
 func (engine *Engine) BuildRootfs(opts *options.BuildRootfsOptions) (containerID string, err error) {
```

Comment:
Why not delete "BuildRootfs"

Reply:
IMO, `BuildRootfs` sounds like it is building rootfs, which is what end users might use when building their own rootfs. I'm not sure whether we need the func in the future. Or, as you annotated before, `BuildRootfs` should be upper to image engine

Reply:
Let's remove it first. We will add it when we need it.

Reply:
removed